README.md

```diff
@@ -24,13 +24,13 @@ The project uses a virtual environment to isolate package dependencies. To creat
 $ poetry install
 ```
 
-You'll also need to clone a new `.env` file from the `.env.tempalate` to store local configuration options. This is a one-time operation on first setup:
+You'll also need to clone a new `.env` file from the `.env.tempalate` to store local configuration options. This is a one-time operation on firgitst setup:
```

Comment:
Accidentally typed "git" into the middle of a word

Reply:
removed